### PR TITLE
Update tutorial to adhere to the Spectro Cloud style guide, and update the hello-universe image tag

### PR DESCRIPTION
**Pull request by sudkul87**
## Describe the Change

This PR removes the word `run` and updates `hello-universe` image tag to `1.0.12`. 

## Review Changes

💻 [Add Preview URL](https://deploy-preview-1271--docs-spectrocloud.netlify.app/registries-and-packs/deploy-pack)

🎫 Jira Ticket: [DOC-724](https://spectrocloud.atlassian.net/browse/DOC-724)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1271#" title="Author self-approved">sudkul87</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — APPROVED** [review]
@sudkul87 can you also update the other tutorial while you're at it please

**Pull request merged**
